modules/task/task.go

```diff
@@ -56,6 +46,12 @@ func GlobalBackend() Backend {
 	return _globalBackend
 }
 
+func globalBackendStatsClient() *statsClient {
```

Comment:
Not sure how Frontend/Backend separation helps with metrics propagation, all of task module code needs access to metrics anyway. But cool with checking this in and iterating on it.